croatian/sljez.md

```diff
@@ -0,0 +1,13 @@
+Jednog jutra, probudis se u cudnoj sobi za sidovima od sljezovih slatkisa.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.